symfony/plugins/orangehrmCorePlugin/Api/V2/Endpoint.php

```diff
@@ -17,50 +17,49 @@
  * Boston, MA  02110-1301, USA
  */
 
-namespace OrangeHRM\Admin\Api\Controller;
+namespace OrangeHRM\Core\Api\V2;
```

Comment:
do we need that `V2` part ?

Reply:
We can remove. But we have to keep v1 as well, So those classes should under `V1`. Since `Endpoint` kind of classes may mess with v1 Endpoints. What's your thoughts?

Reply:
Keep the `V2`